lib/src/install.rs

```diff
@@ -1218,6 +1242,20 @@ pub(crate) async fn install_to_disk(mut opts: InstallToDiskOpts) -> Result<()> {
         loopback_dev.close()?;
     }
 
+    // At this point, all other threads should be gone.
+    if let Some(state) = Arc::into_inner(state) {
+        // If we had invoked `setenforce 0`, then let's re-enable it.
+        match state.selinux_state {
```

Comment:
clippy prefers `if let` here since it's only destructuring a single pattern... but there's also other preexisting things it's complaining about, so I'll just let a few accumulate and do a separate PR to clean them up all at once.